mesh/api/testdata/test_mesh_graph.expected

```diff
@@ -287,20 +287,23 @@
           "nodeType": "infra",
           "healthData": "Healthy",
           "infraData": {
-            "OutboundTrafficPolicy": {
-              "mode": ""
+            "config": {
```

Comment:
Yes I needed to add `revision` but `revision` is only ever present for istiod infra types so `infraData` seemed the best place to put it. Although you can set them to the same value, `revision` and `version` are not the same thing. `revision` is just an arbitrary string value that could be anything the user sets like `canary` or `prod` or whatever. It will probably include the version but doesn't have to. `version` is the semver version of istio and comes directly from istio itself by hitting the `/version` debug endpoint. So I don't think we want to set `version = revision`.

Reply:
Although probably we shouldn't show `revision` in that table? Might want to surface it elsewhere but probably that table should just be the config. I'll update to only show the config.

Speaking of the config, I think we'll just want to show the raw yaml from the istio configmap there? The config object that we show now is an internal structure and it might confuse users if they see values that are not present on their `istio` configmap.

Reply:
I actually don't see a separate rev/config entry in the table:
![Screenshot from 2024-06-20 11-03-43](https://github.com/kiali/kiali/assets/6226732/c27d0633-9bd9-403e-af48-af6ccd6fbd8f)

Were the ui part of these changes built as well?

Reply:
hmm, seems ok.